core/kover/dataset/create.py

```diff
@@ -67,11 +68,33 @@ def _parse_metadata(metadata_path, matrix_genome_ids, warning_callback, error_ca
 	"""
     logging.debug("Parsing metadata.")
     md_genome_ids, md_genome_labels = zip(*(l.split() for l in open(metadata_path, "r")))
-    md_genome_labels = [int(l) for l in md_genome_labels]
-
-    if not np.all(np.unique(md_genome_labels) == [0, 1]):
-        error_callback(Exception("The metadata associated to a genome must be a single binary value (0 or 1)."))
-
+    md_unique_labels, indexes, indices = np.unique(md_genome_labels, return_index=True, return_inverse=True)
```

Comment:
What is the difference between indexes and indices? Both words mean the same thing.